src/dlgConnectionProfiles.cpp

```diff
@@ -133,7 +133,7 @@ void dlgConnectionProfiles::slot_update_description()
     }
 }
 
-void dlgConnectionProfiles::slot_update_website(const QString url)
+void dlgConnectionProfiles::slot_update_website(const QString &url)
```

Comment:
As we have the Qt Copy-on-write process used on this data type does it actually make any difference changing to a reference, given that it is a `const`value anyhow so the local copy in this method will be a *shallow* one to start with?

Reply:
It could be not a shallow one - and clazy likes it this way more anyhow.

Reply:
Okay.